spectrumany.py

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env python3
```

Comment:
This is not a standalone script, it does not need a [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)).